lib/oli_web/views/layout_view.ex

```diff
@@ -111,41 +120,7 @@ defmodule OliWeb.LayoutView do
   end
 
   def account_link(%{:assigns => assigns} = conn) do
```

Comment:
I have three different comments about this function:
1. It used to return a link but now it's a dropdown, so maybe we could change the name accordingly, wdyt?
2. Super nit but we could use the shorthand syntax here (`%{assigns: assigns}`) which I like more 💁🏻‍♂️
3. This could also be a one-line function